Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionCrowdStrikeFalconHost.yaml

```diff
@@ -0,0 +1,282 @@
+      "16400", "Killed", "Deny", "Failure",
+      "32768", "Prevention-blocked", "Deny", "Failure",
+      "32896", "Prevention-blocked,quarantine", "Deny", "Failure",
+      "33024", "Detection", "Allow", "Success",
+      "65536", "Downgraded", "Allow", "Success",
+      "65552", "Prevention-killed", "Deny", "Failure",
+      "65792", "Detection-downgraded", "Allow", "Success",
+      "65808", "Detection-downgraded", "Allow", "Success",
+      "73728", "Detection-downgraded", "Allow", "Success",
+      "73744", "Detection-downgraded", "Allow", "Success",
+      "131088", "Prevention-killed", "Deny", "Failure",
+      "131216", "Prevention-killed,quarantine", "Deny", "Failure",
+      "131584", "Prevention-killed", "Deny", "Failure",
+      "131712", "Prevention-killed,quarantine", "Deny", "Failure",
+      "2099200", "Prevention-blocked", "Deny", "Failure",
+      "2099328", "Prevention-blocked,quarantine", "Deny", "Failure",
+      "4196352", "Prevention-blocked", "Deny", "Failure",
+      "4196480", "Prevention-blocked,quarantine", "Deny", "Failure",
+      "1048576", "Prevention-suspend", "Deny", "Failure",
+      "524288", "Prevention-suspend", "Deny", "Failure",
+      "262144", "Blocking Disabled", "Allow", "Success",
+      "16384", "Safeguard Enabled", "Allow", "Success",
+      "131072", "Kill Failed", "Deny", "Failure",
+      "256", "Policy Disabled", "Allow", "Success",
+      "2097152", "Response Action Already Applied", "Deny", "Failure",
+      "4194304", "Response Failed", "Deny", "Failure"
+  ];
+  let parser = (disabled: bool=false) {
+      let alldata = CommonSecurityLog
+          | where not(disabled)
+              and DeviceVendor == "CrowdStrike"
+              and DeviceProduct == "FalconHost"
+          | where DeviceEventClassID in ("Network Access In A Detection Summary Event", "FirewallMatchEvent");
+      let unparsedfirewalldata = alldata
+          | where DeviceEventClassID == "FirewallMatchEvent"
+          | parse-kv AdditionalExtensions as (deviceId: string, cmdLine: string, connectionDirection: int, eventType: string, hostName: string, icmpCode: int, icmpType: string, localAddress: string, localPort: int, matchCount: int, networkProfile: string, protocol: int, remoteAddress: string, remotePort: int, ruleAction: int, ruleDescription: string, ruleGroupName: string, ruleName: string, status: string) with (pair_delimiter=";", kv_delimiter="=")
+          | extend
+              EventCount = matchCount,
+              EventStartTime = unixtime_milliseconds_todatetime(tolong(ReceiptTime)),
+              EventType = "NetworkSession";
+      let inboundfirewalldata = unparsedfirewalldata
+          | where connectionDirection == 1
+          | extend NetworkDirection = "Inbound"
+          | project-rename
+              SrcIpAddr = localAddress,
+              SrcPortNumber = localPort,
+              DstIpAddr = remoteAddress,
+              DstPortNumber = remotePort;
```

Comment:
For Inbound connections, Localaddress must be the dst address. I think you did the opposite. Please validate.
https://learn.microsoft.com/en-us/azure/sentinel/normalization-schema-network#local-and-remote-aliases

Reply:
Done